app/models/manageiq/providers/openstack/infra_manager.rb

```diff
@@ -16,6 +16,7 @@ class ManageIQ::Providers::Openstack::InfraManager < ::EmsInfra
   include ManageIQ::Providers::Openstack::ManagerMixin
   include HasManyOrchestrationStackMixin
   include HasNetworkManagerMixin
+  include StorageManagerMixin
```

Comment:
I don't think we are doing any linkage from the OpenStack InfraManager to the StorageManager.  I don't think this is there now.